classes/CartRule.php

```diff
@@ -1483,6 +1483,7 @@ public function getAssociatedRestrictions(
 			' . (in_array($type, array('carrier', 'shop')) ? ' AND t.deleted = 0' : '') . '
 			' . ($type == 'cart_rule' ? 'AND t.id_cart_rule != ' . (int) $this->id : '') .
                 $shop_list .
+                ' GROUP BY t.id_' . $type .
```

Comment:
Why is this change needed? I wonder if all the carriers will still be displayed.

Reply:
Without this, the query return duplicate rows.

Reply:
Without this, the query return duplicate rows.

Reply:
As I can't reproduce it, I'd like to keep the previous version.